sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -9146,4 +9146,17 @@ ALTER TABLE testreport_history ADD COLUMN preliminary boolean;
 
 INSERT INTO schema_version (version_number, comment) VALUES (431, '[DEMIS2SORMAS] Handle New Profile: Preliminary Test Results #5551');
 
+-- 2021-12-03 [S2S] Implement outgoing S2S entity validation #7070
+ALTER TABLE areas ADD COLUMN centrally_managed boolean DEFAULT false;
```

Comment:
should probably add them to the history tables as well

Reply:
the history tables are not yet included